torchdrivesim/behavior/iai.py

```diff
@@ -29,21 +37,131 @@ def iai_initialize(location, agent_count, center=(0, 0)):
     return agent_attributes, agent_states, response.recurrent_states
 
 
-def iai_drive(location: str, agent_states: Tensor, agent_attributes: Tensor, recurrent_states: List):
+def iai_conditional_initialize(location, agent_count, agent_attributes=None, agent_states=None, recurrent_states=None, center=(0, 0), traffic_light_state_history=None):
```

Comment:
This should probably also go to `invertedai`.

Reply:
I don't want this function here, but `invertedai` is not quite ready to have this included yet, so let's move it to `torchdriveenv` for the time being, since it's not used anywhere within this repo.